Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -899,6 +890,40 @@ void ABenchmarkGymGameModeBase::ReportAuthoritativePlayerMovement_Implementation
 	CurrentPlayerAvgVelocity = TotalVelocity / TotalPlayers;
 }
 
+void ABenchmarkGymGameModeBase::ReportUserExperience_Implementation(const FString& WorkerID, float RTTime, float UpdateTime)
+{
+	check(HasAuthority());
+
+	if (!WorkerID.IsEmpty())
+	{
+		LatestClientUXMap.Emplace(WorkerID, UX{RTTime, UpdateTime});
+	}
+
+	AveragedClientRTTMS = 0.f;
+	AveragedClientUpdateTimeDeltaMS = 0.f;
+
+	for (auto& Entry : LatestClientUXMap)
+	{
+		AveragedClientRTTMS = FMath::Max(AveragedClientRTTMS, Entry.Value.RTT);
```

Comment:
I am probably missing something here but... AveragedClientRTTMS seems to be set to the maximum of all workers ClientRTTMS instead of the average?

Reply:
Yeah I'll agree it's confusing. So each server worker averages it's client worker ux metrics, and sends that as a single value to the auth game mode. The auth game mode then looks for the worst aggregated average, from any one workers perspective, and if it's worse than the threshold, fails the test. I'll update the variable naming to try to make that clearer.

Reply:
Perhaps AveragedClientRTTMS should be WorstAveragedClientRTTMS